chromadb/test/conftest.py

```diff
@@ -30,16 +30,80 @@
 from chromadb.types import SeqId, OperationRecord
 from chromadb.api.client import Client as ClientCreator
 
+VALID_PRESETS = ["fast", "normal", "slow"]
+CURRENT_PRESET = os.getenv("PROPERTY_TESTING_PRESET", "normal")
+
+if CURRENT_PRESET not in VALID_PRESETS:
+    raise ValueError(
+        f"Invalid property testing preset: {CURRENT_PRESET}. Must be one of {VALID_PRESETS}."
+    )
+
 hypothesis.settings.register_profile(
-    "dev",
+    "base",
     deadline=45000,
     suppress_health_check=[
         hypothesis.HealthCheck.data_too_large,
         hypothesis.HealthCheck.large_base_example,
         hypothesis.HealthCheck.function_scoped_fixture,
     ],
 )
-hypothesis.settings.load_profile(os.getenv("HYPOTHESIS_PROFILE", "dev"))
+
+hypothesis.settings.register_profile(
+    "fast", hypothesis.settings.get_profile("base"), max_examples=50
+)
+# Hypothesis's default max_examples is 100
+hypothesis.settings.register_profile(
+    "normal", hypothesis.settings.get_profile("base"), max_examples=100
+)
+hypothesis.settings.register_profile(
+    "slow", hypothesis.settings.get_profile("base"), max_examples=200
```

Comment:
it's a little hard to prescribe good defaults here for max_examples, we can adjust as needed